### PR TITLE
fix(listview): Improve focused and selected state appearance of listview in accordance with design

### PR DESCRIPTION
**Pull request by joneff**
Fixes https://github.com/telerik/kendo-theme-default/issues/434

### CODE REVIEW

**Comment by joneff** [comment]
Please close related item as well

File: packages/bootstrap/scss/listview/_theme.scss [hunk 1]
```diff
@@ -1,9 +1,16 @@
-@include exports('listview/theme') {
-    .k-listview > .k-state-focused {
-        box-shadow: inset 0 0 0 3px try-darken( $node-bg, 7 );
+@include exports( "listview/theme" ) {
```
Comment (gyoshev) [inline packages/bootstrap/scss/listview/_theme.scss]: This file is identical to the default theme file, so I'd suggest importing the default theme, in order to reduce code duplication.

File: packages/material/scss/listview/_theme.scss [hunk 2]
```diff
@@ -1 +1,19 @@
...
+
+        > .k-state-focused {
+            background-color: rgba(0, 0, 0, .07);
+        }
+
+        > .k-state-selected {
+            color: inherit;
+            background-color: rgba(0, 0, 0, .04);
+        }
+        > .k-state-selected.k-state-focused {
```
Comment (gyoshev) [inline packages/material/scss/listview/_theme.scss]: It seems that if the states are declared like

```scss
.k-state-selected {}
.k-state-focused {}
```

There won't be the need for a combined selector.

Comment (joneff) [reply packages/material/scss/listview/_theme.scss]: My bad. They were supposed to be different.

File: packages/bootstrap/scss/listview/_theme.scss (hunk 1 above)
Comment (joneff) [reply packages/bootstrap/scss/listview/_theme.scss]: Done

**Pull request merged**